.github/workflows/test.yaml

```diff
@@ -171,20 +144,6 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 20
-      - name: az CLI login
-        if: steps.gen-id.outputs.RUN_TEST == 'true' && matrix.credential == 'azure'
```

Comment:
perhaps needed for https://github.com/radius-project/samples/blob/edge/samples/dapr/dapr-azure.bicep#L110